src/TextHelpers.js

```diff
@@ -186,5 +194,5 @@ export const formatDatePair = (startTime, endTime, returnAsObj) => {
 }
 
 export function isDateInTheFuture (date) {
-  return moment(date).isAfter(moment())
+  return DateTime.fromMillis(date).isAfter(DateTime.now())
```

Comment:
This function previously was able to accept Dates in many different formats, but is now ONLY capable of accepting timestamps.

Reply:
there's no way to create a DateTime from different formats? lets just make sure this isn't a problem where we use it of course